catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -311,6 +317,51 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
     //
     // Private helper methods
     //
+    private DeleteRequest rewriteQuery(DeleteRequest deleteRequest)
```

Comment:
`rewriteRequestToAvoidHistoryConflicts`?  If you want, you might be able to get it shorter by including helpful JavaDoc.